### PR TITLE
Docs: Update export-pdf.md

### PR DESCRIPTION
**Pull request by oddlittlebird**
Minor edits and one question.

**Special notes for your reviewer**: Where is the PDF saved and how is the file named?

### CODE REVIEW

File: docs/sources/enterprise/export-pdf.md [hunk 1]
```diff
@@ -10,15 +10,10 @@ weight = 900
...
 > Only available in Grafana Enterprise v6.7+.
 
-## Generating dashboard PDF
-
-Dashboard can be exported to PDF from **Share dashboard** menu.
-
-1. Click on the share icon in the dashboard menu.
-1. Go to PDF tab.
-1. Select the layout option for exported dashboard. By default it is set to **Portrait** but it's also possible to select **Landscape**.
+1. Click the title of the dashboard that you want to export as PDF and then click **Share**.
```
Comment (alexanderzobnin) [inline docs/sources/enterprise/export-pdf.md]: That's not true it seems. Share modal available from share button, not dashboard title.
Screenshot from 2020-03-13 12-51-00

File: docs/sources/enterprise/export-pdf.md [hunk 2]
```diff
@@ -10,15 +10,10 @@ weight = 900
...
+1. On the PDF tab,sSelect the layout option for exported dashboard: **Portrait** or **Landscape**.
```
Comment (alexanderzobnin) [inline docs/sources/enterprise/export-pdf.md]:
```suggestion
1. On the PDF tab, select the layout option for exported dashboard: **Portrait** or **Landscape**.
```

**Review by alexanderzobnin — CHANGES_REQUESTED** [review]

**Comment by alexanderzobnin** [comment]
> **Special notes for your reviewer**: Where is the PDF saved and how is the file named?
PDF will be opened in a new tab in the browser, then you can save it from the browser. File name matches dashboard id, but I think we should fix it and use PDF title or something more human-readable.

**Review by alexanderzobnin — APPROVED** [review]
Looks good now!

**Comment by oddlittlebird** [comment]
@alexanderzobnin , the build-pipeline is still yellow after several hours. Any idea why?

**Pull request merged**